common/test/acceptance/pages/studio/index.py

```diff
@@ -28,18 +29,22 @@ def course_runs(self):
     def has_processing_courses(self):
         return self.q(css='.courses-processing').present
 
-    def create_rerun(self, display_name):
+    def create_rerun(self, course_key):
         """
-        Clicks the create rerun link of the course specified by display_name.
+        Clicks the create rerun link of the course specified by course_key
         """
-        name = self.q(css='.course-title').filter(lambda el: el.text == display_name)[0]
-        name.find_elements_by_xpath('../..')[0].find_elements_by_class_name('rerun-button')[0].click()
+        actions = ActionChains(self.browser)
```

Comment:
Please add a comment that explains what you are doing here and why. E.g. "The 'Re-run course' link doesn't show up until you mouseover the course in the listing."